js/DailyDouble.js

```diff
@@ -0,0 +1,22 @@
+if (typeof require !== 'undefined') { 
+  const Question = require('./Question.js');
+};
+
+class DailyDouble extends Question {
+  constructor(clue) {
+    super(clue)
+  }
+
+  validateWager(wager, currentPlayer) {
+    if (isNaN(wager) || wager > currentPlayer.points || wager < 5) {
+      return false;
+    } else {
+      return true;
+    };
```

Comment:
I'm not sure that leaving semicolon's like this at the end of block scope curly braces is correct styling -- lets run these through a linter and remerge

Reply:
Right on that count!

`{` and `}` are meant to begin and close a group of statements so semi-colons shouldn't be used.